sdk/lib/src/imagekitio-angular/ik-video/ik-video.component.ts

```diff
@@ -0,0 +1,81 @@
+import { Component, AfterViewInit, OnInit, ElementRef, Input, OnChanges } from '@angular/core';
+import { ImagekitService } from '../imagekit.service';
+
+@Component({
+  selector: 'ik-video',
+  template: `<video controls><source src={{src}}>`,
+})
+
+export class IkVideoComponent implements OnInit {
+    @Input('urlEndpoint') urlEndpoint:string;
+    @Input('path') path:string; //required
+    @Input('src') src:string; //required
+    @Input('transformation') transformation:Array<Object> = [];
+    @Input('transformationPosition') transformationPosition:string;
+    @Input('queryParameters') queryParameters:Object;
```

Comment:
Possibly can be refined a bit like here: https://github.com/imagekit-developer/imagekit-react/blob/master/src/components/IKVideo/props.ts#L6

Reply:
The typescripting will be implemented in a separate PR